symfony/phpunit-bridge/3.3/phpunit.xml.dist

```diff
@@ -5,16 +5,16 @@
          xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.1/phpunit.xsd"
          backupGlobals="false"
          colors="true"
-         bootstrap="vendor/autoload.php"
+         bootstrap="tests/bootstrap.php"
```

Comment:
should we update phpunit bridge configuration too ? or should this be done in phpunit bridge script instead?